pkgs/development/libraries/libheif/default.nix

```diff
@@ -22,20 +23,20 @@
 
 stdenv.mkDerivation rec {
   pname = "libheif";
-  version = "1.15.2";
+  version = "1.17.1";
```

Comment:
1.17.3 is now out

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/254708/commits/de0104d374dd04034686b32a29aaaf004b3ef3bc